src/commands/secret/addSecret/SecretNameStep.ts

```diff
@@ -29,8 +29,9 @@ export class SecretNameStep extends AzureWizardPromptStep<ISecretContext> {
             return validateUtils.getInvalidLengthMessage();
         }
 
-        if (!validateUtils.isLowerCaseAlphanumericWithSymbols(value)) {
-            return validateUtils.getInvalidLowerCaseAlphanumericWithSymbolsMessage();
+        const symbols: string = '-';
```

Comment:
I might be crazy, but because of how the property is named, I keep accidentally thinking that this is turning on a rule that _allows_ symbols to repeat, not that it's checking against it.

Reply:
Wait maybe I'm not crazy? But in that case, I think we shouldn't default `canSymbolsRepeat` to false.

Reply:
> I might be crazy, but because of how the property is named, I keep accidentally thinking that this is turning on a rule that allows symbols to repeat, not that it's checking against it.

Not crazy, that's exactly what it does!

What are you thinking?  Default `canSymbolsRepeat` to true? Or rename the option to something else?

I chose to have this be the default behavior because it is probably the more common/standard behavior for Azure resource naming, and I like passing in the least amount of vars to accomplish the more default/common behavior.

Reply:
Based on our discussion, it looks like there is a lot of non-containerApp resources that do allow repeating symbols as default, so making the default behavior true instead of false.